js/ui/scroll_view/ui.scrollable.js

```diff
@@ -550,7 +550,7 @@ const Scrollable = DOMComponent.inherit({
         const elementPosition = elementPositionRelativeToContent - pushBackOffset;
         const elementSize = $element[isVertical ? 'outerHeight' : 'outerWidth']();
         const scrollLocation = (isVertical ? this.scrollTop() : this.scrollLeft());
-        const clientSize = (isVertical ? this.clientHeight() : this.clientWidth());
+        const clientSize = this._container().get(0)[isVertical ? 'clientHeight' : 'clientWidth'];
```

Comment:
imho we need a new test in this PR: 
https://codepen.io/DanIgnatov/pen/LYZXVoB?editors=1010 - both the horizontal and vertical scroll bar sizes are not counted in FF

Reply:
Tests was added in https://github.com/DevExpress/DevExtreme/pull/15175/commits/902b7584e4fe955a79eece101cc58e0704092cdd